### PR TITLE
[4] [UI] Show logo with animation in Login page

### PR DESCRIPTION
**Pull request by manh-t**
https://github.com/manh-t/flutter-survey/issues/4

## What happened 👀

- Show Nimble logo at center of screen with opacity animation, then transform it to be top of screen.
- Do not show Login form for now.

## Proof Of Work 📹

iOS:

Android:

https://user-images.githubusercontent.com/60863885/116396483-a9d31d80-a84f-11eb-9347-dd5a33857caa.mp4

### CODE REVIEW

File: lib/resouces/assets.dart [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Assets {
+  const Assets._();
+
+  static const String imgLoginBackground = "assets/images/login_background.jpg";
+  static const String icNimbleLogo = "assets/icons/ic_nimble_logo.svg";
+}
```
Comment (markgravity) [inline lib/resouces/assets.dart:6]: You can use [flutter_gen](https://pub.dev/packages/flutter_gen) as a code generator for your assets, fonts, colors,.. …

**Review by markgravity — APPROVED** [review]
LGTM 🤖

File: lib/resouces/assets.dart (hunk 1 above)
Comment (manh-t) [reply lib/resouces/assets.dart:6]: @markgravity thanks for the suggestion, these assets are simple so I don't use code gen for them 😁

File: lib/resouces/assets.dart [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Assets {
+  const Assets._();
+
+  static const String imgLoginBackground = "assets/images/login_background.jpg";
```
[the remark this answers is not in the document]
Comment (manh-t) [reply lib/resouces/assets.dart:4]: oh my bad, will update 🙏

[the remark this answers is not in the document]
Comment (manh-t) [reply lib/resouces/assets.dart]: @luongvo updated 🚀

**Review by luongvo — APPROVED** [review]
lgtm

**Pull request merged**